### PR TITLE
Add Datadog and more logos (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Follow up https://github.com/ruby/www.ruby-lang.org/pull/3178, https://github.com/ruby/www.ruby-lang.org/pull/3180

### CODE REVIEW

File: ko/about/website/index.md [hunk 1]
```diff
@@ -36,13 +36,33 @@ lang: ko
...
+
+[Fastly][fastly] (CDN)
+
+<img src="../../../images/sponsor/fastly.png" alt="Fastly" width="200" height="200" />
+
+[Hatena][hatena] ([mackerel][mackerel], 서버 모니터링)
+
+<img src="../../../images/sponsor/mackerel.png" alt="mackerel" width="300" height="80" />
+
+[Datadog][datadog] (server monitoring)
```
Comment (marocchino) [inline ko/about/website/index.md:63]:
```suggestion
[Datadog][datadog] (서버 모니터링)
```

Comment (riseshia) [reply ko/about/website/index.md:63]: 🙈

Comment (riseshia) [reply ko/about/website/index.md]: fixed [69b9e42](https://github.com/ruby/www.ruby-lang.org/pull/3181/commits/69b9e42bbf7e18d81396616f74c8532bbf30212c)

**Review by marocchino — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks~!

**Pull request merged**